src/components/MediaContent.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Constants from "../constants/Constants";
+
+interface Props {
+  question: object;
+}
+
+const MediaContent = ({ question }: Props) => {
+  const getEmbedLink = (mediaContentLink: string) => {
+    if (mediaContentLink.includes("/view?")) {
+      const view: string = mediaContentLink.substring(
+        mediaContentLink.indexOf("/view?")
```

Comment:
This is to avoid to have to make complex manipulation when getting the url. With this function `getEmbedLink`, all you have to do is click on "get the link" (sorry, my google drive is in french :/ ): 

![image](https://user-images.githubusercontent.com/57750426/166706563-1a5a1715-b1af-41c3-bfe8-fea064099270.png)


Without `getEmbedLink` you have to:

1. Double click on the image to open it
2. Click the options, and click open in a new tab

![image](https://user-images.githubusercontent.com/57750426/166707156-a1f5f286-8669-4940-9e26-ee9d278444ac.png)

3. Click the options again, and click "get embed link"

![image](https://user-images.githubusercontent.com/57750426/166707453-a56b6e71-88d2-493e-abef-31813f189cdb.png)

4. Get the link

![image](https://user-images.githubusercontent.com/57750426/166707731-0bd83226-91d3-41a2-b5f1-6e22b5705b55.png)

Reply:
@blcham If you think this manipulation is not a problem, I can get rid of the method.